packages/expo-updates/android/src/main/java/expo/modules/updates/procedures/StateMachineSerialExecutorQueue.kt

```diff
@@ -85,9 +85,4 @@ class StateMachineSerialExecutorQueue(
       }
     }
   }
-
-  fun cancel() {
-    executorJob.cancel()
-    procedureChannel.close()
-  }
```

Comment:
just to double confirm if that's an intentional change

Reply:
Yes, that should have been part of the downstack PR